cg/apps/tb/add.py

```diff
@@ -39,145 +40,196 @@ def add_analysis(cls, config_stream):
-                bai_path = bam_path.replace('.bam', '.bai')
-
-            data.append({
-                'path': bam_path,
-                'tags': ['bam', sample_data['id']],
-                'archive': False,
-            })
-            data.append({
-                'path': bai_path,
-                'tags': ['bam-index', sample_data['id']],
-                'archive': False,
-            })
+                data.append({"path": vcf_path, "tags": [vcf_tag], "archive": True})
+                data.append(
+                    {
+                        "path": f"{vcf_path}.tbi"
+                        if variant_type == "snv"
+                        else f"{vcf_path}.csi",
+                        "tags": [f"{vcf_tag}-index"],
+                        "archive": True,
+                    }
+                )
+
+        for sample_data in sampleinfo_data["samples"]:
+            data.append(
+                {
+                    "path": sample_data["sambamba"],
+                    "tags": ["coverage", sample_data["id"]],
+                    "archive": False,
+                }
+            )
+
+            # Cram pre-processing
+            cram_path = sample_data["cram"]
+            crai_path = f"{cram_path}.crai"
+            if not Path(crai_path).exists():
+                crai_path = cram_path.replace(".cram", ".crai")
+
+            data.append(
+                {
+                    "path": cram_path,
+                    "tags": ["cram", sample_data["id"]],
+                    "archive": False,
+                }
+            )
+            data.append(
+                {
+                    "path": crai_path,
+                    "tags": ["cram-index", sample_data["id"]],
+                    "archive": False,
+                }
+            )
 
             # Only for wgs data
             # Downsamples MT bam pre-processing
-            if sample_data['subsample_mt']:
-                mt_bam_path = sample_data['subsample_mt']
+            if sample_data["subsample_mt"]:
+                mt_bam_path = sample_data["subsample_mt"]
                 mt_bai_path = f"{mt_bam_path}.bai"
                 if not Path(mt_bai_path).exists():
-                    mt_bai_path = mt_bam_path.replace('.bam', '.bai')
-                data.append({
-                    'path': mt_bam_path,
-                    'tags': ['bam-mt', sample_data['id']],
-                    'archive': False,
-                })
-                data.append({
-                    'path': mt_bai_path,
-                    'tags': ['bam-mt-index', sample_data['id']],
-                    'archive': False,
-                })
-
-            cytosure_path = sample_data['vcf2cytosure']
-            data.append({
-                'path': cytosure_path,
-                'tags': ['vcf2cytosure', sample_data['id']],
-                'archive': False,
-            })
+                    mt_bai_path = mt_bam_path.replace(".bam", ".bai")
```

Comment:
Bam?

Reply:
yes, that file is downsampled to 150 x and only cover the 16 kb of the MT so it is rather small. No need to convert it in the workflow to CRAM